website/static/js/fangorn.js

```diff
@@ -223,6 +223,9 @@ function resolveIconView(item) {
         return returnView('link');
     }
     if (item.data.nodeType === 'project') {
+        if (item.data.parentIsFolder && item.data.isFolder && item.data.isPointer) {
```

Comment:
I looked at what was in the data for the collections that were created and checked for those settings when returning the icon. I don't know why they are set like that.